articles/200_migration_guide_from_ros1.md

```diff
@@ -113,7 +145,7 @@ Apply the following changes to use `ament_cmake` instead of `catkin`:
   * `CATKIN_GLOBAL_LIBEXEC_DESTINATION`: `lib`
   * `CATKIN_GLOBAL_SHARE_DESTINATION`: `share`
 
-  * `CATKIN_PACKAGE_BIN_DESTINATION`: `lib/${PROJECT_NAME}`
+  * `CATKIN_PACKAGE_BIN_DESTINATION`: `bin/${PROJECT_NAME}`
```

Comment:
I don't think that's right. Maybe `bin`, but certainly not `bin/${PROJECT_NAME}`.